### DISCUSSION TITLE
i lost my telegram account

### DISCUSSION THREAD

**Discussion by Hzone-glitch** [masterking32/MasterHamsterKombatBot#446] [General]
Hi 
I lost my telegram account😭 but I have bot connect to my account and my wallet is connected too
Please🥹 add new ability to click on **set as default withdrawal option ** with Bot.

Thank you for your awesome Bot🙏

**Comment by Fy0urM** [comment]
I doubt it will help you, the default withdrawal can be changed.

![Screenshot_20240914-214654055_1_1.jpg](https://github.com/user-attachments/assets/34851093-c4cd-4a93-85c8-fb5257b26540)

**Comment by Hzone-glitch** [comment]
> I doubt it will help you, the default withdrawal can be changed.
> 
> ![Screenshot_20240914-214654055_1_1.jpg](https://github.com/user-attachments/assets/34851093-c4cd-4a93-85c8-fb5257b26540)
> 
> 

Yes thats right 👍🏽 (but I don't want to reset)

Sorry my English is bad.

I cannot open my telegram account (my telegram is ban my phone because I buy and use virtual phone number !! )

And hamster is connect to bot and it's works well.

I want to choose**on-chain airdrop**  with the robot.

**Comment by Fy0urM** [comment]
i understand u prev comment

**Comment by Hzone-glitch** [comment]
> Hi 
> I lost my telegram account😭 but I have bot connect to my account and my wallet is connected too
> Please🥹 add new ability to click on **set as default withdrawal option ** with Bot.
> 
> Thank you for your awesome Bot🙏

![IMG_20240914_155954.jpg](https://github.com/user-attachments/assets/df5d0058-2f84-4aad-a99a-0cbdc709d820)

**Comment by Fy0urM** [comment]
oh, i just checked now, the game's servers are sick again, http 500 internal server error. need to try again later

**Comment by Fy0urM** [comment]
> oh, i just checked now, the game's servers are sick again, http 500 internal server error. need to try again later

,

**Comment by Hzone-glitch** [comment]
> > oh, i just checked now, the game's servers are sick again, http 500 internal server error. need to try again later
> 
> ,

Ok😅 I will check it later
If it's work I donate half of hamsters token
Probably not a lot of money, but it's all I can do.

**Comment by Fy0urM** [comment]
Great, I wish you the best of luck.

**Comment by Hzone-glitch** [comment]
> Great, I wish you the best of luck.

You are a great man 🔥
I wish I could make up for it🔮

**Comment by tboy1337** [comment]
@masterking32 why was the code and comments deleted?

**Comment by Fy0urM** [comment]
> @masterking32 why was the code and comments deleted?

Because I asked

**Comment by tboy1337** [comment]
@Fy0urM was there something wrong with the code? I was going to use it.

**Comment by Fy0urM** [comment]
> @Fy0urM was there something wrong with the code? I was going to use it.

Yes, of course, if you already have one. But then I'm asking you not to publish it please.

**Comment by tboy1337** [comment]
Is it because it could be used to steal accounts or something?

**Comment by Fy0urM** [comment]
> Is it because it could be used to steal accounts or something?

something like that.

**Comment by Gantar15** [comment]
> > Is it because it could be used to steal accounts or something?
> 
> something like that.

Can you share the code again pls?

**Comment by Fy0urM** [comment]
> Can you share the code again pls?

i'm sorry, but no.
why not? because i've decided to.

**Comment by tboy1337** [comment]
@Fy0urM I kinda get the reasoning but imo the code is nowhere near being able to steal accounts.  It can only confirm the address that has previously been entered and it cant even grab that address, you have to enter it manually so good luck trying to take over accounts with that, you would need full access to the account in the first place so you might as well steal it manually.  I guess the code could be modified to do a good job of stealing accounts but you would have to know how to code in the first place so you could probably write it yourself then lol.  Thinking about it, for me it doesn’t make sense to use this code as i don’t know the wallet addresses of all my accounts so i will need to manually log them in anyway to find the address and therefore might as well do it manually as it will be quicker.

**Comment by Fy0urM** [comment]
> @Fy0urM I kinda get the reasoning but imo the code is nowhere near being able to steal accounts.  It can only confirm the address that has previously been entered and it cant even grab that address, you have to enter it manually so good luck trying to take over accounts with that, you would need full access to the account in the first place so you might as well steal it manually.  I guess the code could be modified to do a good job of stealing accounts but you would have to know how to code in the first place so you could probably write it yourself then lol.  Thinking about it, for me it doesn’t make sense to use this code as i don’t know the wallet addresses of all my accounts so i will need to manually log them in anyway to find the address and therefore might as well do it manually as it will be quicker.

If you think I'm a paranoid psychopath, that's your right. The code I created has been tested by me. I didn't say you can steal an account, but you can do something different. Knowing what I know. Just use your head and maybe you'll figure it out. I'll tell you one thing, the developers of Hamster Kombat were probably hired by McDonald's.

upd: anyway, I don't intend to discuss this in public.

**Comment by autoattack999** [comment]
> Hi I lost my telegram account😭 but I have bot connect to my account and my wallet is connected too Please🥹 add new ability to click on **set as default withdrawal option ** with Bot.
> 
> Thank you for your awesome Bot🙏

hello, can you help me, my problem is the same as yours. I need the code, can you share it with me? I need it, help me :(

**Comment by Fy0urM** [comment]
@tboy1337 If you want to know what I'm talking about, you can give me your bearer on telegram and I'll show you.

**Reply by autoattack999** [reply]
hello friend, can you share the code with me, because i lost my telegram, can you share it via private, please help me :(

you can send code via discord. my dc : auto_attack

please help me @Fy0urM

**Reply by Gantar15** [reply]
@Fy0urM If you don't want to share the code publicly, why don't you share it with me privately because I really need it? I would be very grateful) my discord: feedgq
